packages/presentation/lib/screens/home/widgets/rating.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+
+enum Mode { base, full }
+
+class Rating extends StatelessWidget {
+  static const minNewRating = 0;
+  static const maxNewRating = 5;
+
+  late final double normalizedRating;
```

Comment:
try to avoid late. https://www.youtube.com/watch?v=0nXXUzMyF8c - link for kotlin, but the same logic in dart

Reply:
Fixed